llvm/test/DebugInfo/NVPTX/packed_bitfields.ll

```diff
@@ -16,7 +16,7 @@
 ; CHECK-NEXT: .b8 1    // DW_AT_byte_size
 ; CHECK-NEXT: .b8 6    // DW_AT_bit_size
 ; Negative offset must be encoded as an unsigned integer.
-; CHECK-NEXT: .b64 0xffffffffffffffff // DW_AT_bit_offset
+; CHECK-NEXT: .b8 127  // DW_AT_bit_offset
```

Comment:
127  (0x7f) does not look like a negative number. Do we really expect to see 127 as representation of some *negative* value?

Reply:
I believe DW_FORM_sdata encodes its value as a signed LEB128, so this is a one-byte encoding of -1.